README.md

```diff
@@ -118,6 +118,66 @@ function ExampleWrapper({
 
 [Try it on Code Sandbox](https://codesandbox.io/s/x70ly749rq)
 
+## Advanced usage
+
+Some use-cases need the cached rows to be cleared for example a sorted list. Otherwise the InfiniteLoader won't call the function passed in `loadMoreItems` prop. This can be achieved by calling the `InfiniteLoader` component's method `resetloadMoreItemsCache`.
```

Comment:
This wording seems a little confusing. Maybe something like?

> Some use cases require cached items to be reset. For example, after a list has been sorted, previously cached items may be invalid. You can let `InfiniteLoader` know that it needs to reload cached items by calling the `resetLoadMoreItemsCache` method.

Reply:
Much more clear, thanks!